src/material/core/tokens/_token-utils.scss

```diff
@@ -93,6 +93,19 @@ $_component-prefix: null;
   }
 }
 
+// Returns the name of a token including the current prefix. Intended to be used in calculations
+// involving tokens. `create-token-slot` should be used when outputting tokens.
+@function create-token-variable($token) {
```

Comment:
can we call this `get-token-variable-name` or maybe just `get-token-variable` if you think thats descriptive enough? The `get-` terminology sounds more function-y where `create-` sounds more mixin-y